packages/flutter/test/material/popup_menu_test.dart

```diff
@@ -3785,6 +3785,119 @@ void main() {
     await tester.pumpAndSettle();
     expect(count, 1);
   });
+
+  testWidgetsWithLeakTracking('animationDuration works as expected in showMenu', (WidgetTester tester) async {
+    final Key buttonKey = UniqueKey();
+    await tester.pumpWidget(
+      MaterialApp(
+        home: Builder(
+          builder: (BuildContext context) {
+            return Scaffold(
+              body: Center(
+                child: ElevatedButton(
+                  key: buttonKey,
+                  child: const Text('Tap'),
+                  onPressed: () {
+                    showMenu(
+                      context: context,
+                      animationDuration: const Duration(milliseconds: 400),
+                      position: const RelativeRect.fromLTRB(0.0, 0.0, 15.0, 15.0),
```

Comment:
```suggestion
                      position: RelativeRect.fill,
```

Use something like for the position as we're testing transition duration here.